docs/mkdocs-base.yml

```
@@ -46,7 +46,8 @@ nav:
                   - Guide:
                           - user/guide/index.md
                   - Manual :
-                          - user/manual/index.md
+                          - Introduction: user/manual/index.md
+                          - Profile Page: user/manual/profile-page.md
```

The index is already referenced and doesn't need a title per se so that if a user clicks on `manual` it automatically will link to the page.

```suggestion
                          - user/manual/index.md
                          - Profile Page: user/manual/profile-page.md
                          - Site Pages: user/manual/site-pages.md
```